### DISCUSSION TITLE
Disable linter for some files.

### DISCUSSION THREAD

**Discussion by mput** [clj-kondo/clj-kondo#1636] [Q&A] [answered]
Does it possible somehow to disable some linters for files by wildcard?
Something like this:
```
{:linters {:inline-def {:level :off
                        :ns ["*-test"]}}}}
```

**Comment by borkdude** [comment, marked as answer]
I believe this issue is about that: https://github.com/clj-kondo/clj-kondo/issues/1618
Feel free to comment and upvote the issue.
